src/main/webapp/vendor/startbootstrap/index.html

```diff
@@ -129,12 +130,52 @@
             <h1 class="h3 mb-0 text-gray-800">Scheduler</h1>
           </div>
 
-          <!-- Content Row -->
           <div class="row">
-            <!-- Content Column -->
-            <div class="col-lg-6 mb-4"></div>
+            <!-- Course Selection Area -->
+            <div class="col-xl-8 col-lg-7">
+              <div class="card shadow mb-4">
+                <!-- Card Header - Dropdown -->
+                <div class="card-header py-3 d-flex flex-row align-items-center justify-content-between">
+                  <h6 class="m-0 font-weight-bold text-primary">Choose a Course:</h6>
+                </div>
+                <!-- Card Body -->
+                <div class="card-body">
+                  <form onsubmit="return false">
+                    <div class="form-row">
+                      <div class="form-group col-md-10">
+                        <select id="courses" class="form-control"></select>
+                      </div>
+                      <div class="form-group col-md-2 text-center">
+                        <button onclick="addToSelected()" class="rounded-circle border-0 btn-lg"><i class="fas fa-plus"></i></button>
+                      </div>
+                    </div>
+                  </form>
+                </div>
+              </div>
+            </div>
+             <!-- Pie Chart -->
+            <div class="col-xl-4 col-lg-5">
+              <div class="card shadow mb-4">
+                <!-- Card Header - Dropdown -->
+                <div class="card-header py-3 d-flex flex-row align-items-center justify-content-between">
+                  <h6 class="m-0 font-weight-bold text-primary">Selected Courses</h6>
+                </div>
+                <!-- Card Body -->
+                <div class="card-body">
+                  <form>
+                    <ul class="list-group" id="selectedClasses"></ul>
```

Comment:
ids should use dashes, not camel case: `selected-classes`

https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming